docs/grpc/c#.md

```diff
@@ -89,7 +112,7 @@ Console.WriteLine(response);
 var request = new InvoiceSubscription();
 using (var call = client.SubscribeInvoices(request))
 {
-    while (await call.ResponseStream.MoveNext())
+    while (await call.ResponseStream.MoveNext(cancellationToken))
```

Comment:
nit: where is this coming from? Copy/paste error?

Reply:
VSCode was complaining that it needed a cancellationToken. It had a red squiggly. I see now that it indeed didn't need one. Removed it again...

Reply:
If the IDE/linter recommends using a cancellation token, then we could maybe add it. It just needs to be introduced in the example code, otherwise someone copy/pasting the code will get an error because the variable is not defined (at least I think so, I don't have an environment to test this out).

Reply:
There's not really a neat way to introduce this cancellationToken in the example code here. I'd have to wrap it in a method and that's just distracting I think. All editors will recommend you to use a cancellationToken if there is one passed into the method you're doing the call from. So this is probably the best option. No copy/paste errors and no distractions.